src/lib/routes/admin-api/instance-admin.ts

```diff
@@ -84,11 +84,11 @@ class InstanceAdminController extends Controller {
         return {
             OIDCenabled: true,
             SAMLenabled: false,
-            clientApps: [
-                { range: 'allTime', count: 15 },
-                { range: '30d', count: 9 },
-                { range: '7d', count: 5 },
-            ],
+            clientApps: {
+                allTime: 15,
+                '30d': 9,
+                '7d': 5,
+            },
```

Comment:
Is the frontend prepared for this change?

Reply:
If we use this in the frontend though.

Reply:
[d6fa7cf](https://github.com/Unleash/unleash/pull/6155/commits/d6fa7cffeb017972ed59bf8cd052340486ece512) should fix this